Sources/ScrollingTabController.swift

```diff
@@ -202,17 +200,62 @@ open class ScrollingTabController: UIViewController, UIScrollViewDelegate, UICol
     }
 
     func configureViewControllers() {
+        for viewController in viewControllers {
+            items.append(TabItem(addTabContainer(), viewController))
+        }
+    }
+
+    public func injectInitialViewControllers(_ viewControllers: [UIViewController]) {
+        self.viewControllers = viewControllers
+        guard tabControllersView != nil else { return }
+
         for item in items {
             let child = item.controller
             child.willMove(toParentViewController: nil)
             child.view.removeFromSuperview()
             child.removeFromParentViewController()
             item.container.removeFromSuperview()
         }
-        
-        for viewController in viewControllers {
-            items.append(TabItem(addTabContainer(), viewController))
+        items = []
+
+        configureViewControllers()
+    }
+
+    public func appendViewController(_ newViewControllers: [UIViewController]) {
```

Comment:
Are you talking about 
```
for item in items {
            let child = item.controller
            child.willMove(toParentViewController: nil)
            child.view.removeFromSuperview()
            child.removeFromParentViewController()
            item.container.removeFromSuperview()
}
```?

I don't think that needs to happen in appending right? Cause we aren't getting rid of the old items.

And then for removing, yes I should add that.